unifi_mqtt/cli.py

```diff
@@ -79,13 +80,23 @@ def main(
         password=mqtt_password,
     )
 
+    # enable unsafe mode for aiohttp's ClientSession CookieJar if unifi_host is IP address instead of FQDN
+    try:
+        # check if unifi_host is a valid IP address
+        netaddr.IPAddress(unifi_host)
```

Comment:
would ipaddress work under python3? https://docs.python.org/3/library/ipaddress.html

Reply:
Yes, that would work, thanks! I'll replace it - saves us an external dependency.